imap_processing/tests/cdf/test_utils.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import xarray as xr
+
+from imap_processing.cdf.global_attrs import ConstantCoordinates
+from imap_processing.cdf.utils import calc_start_time, write_cdf
+from imap_processing.swe.swe_cdf_attrs import swe_l1a_global_attrs
+
+
+def test_calc_start_time():
+    # TODO: Update this when launch time is updated
+    launch_time = np.datetime64("2010-01-01T00:01:06.184")
+    assert calc_start_time(0) == launch_time
+    assert calc_start_time(1) == launch_time + np.timedelta64(1, "s")
+
+
+def test_write_cdf(tmp_path):
+    # Set up a fake dataset
+    # lots of requirements on attributes, so depend on SWE for now
+    ds = xr.Dataset(
```

Comment:
```suggestion
    dataset = xr.Dataset(
```

Reply:
I updated this and `fname` to spell it out. But, I think there is an argument to be made for keeping it `ds` for xarray dataset, `df` for pandas dataframe, `np` for numpy etc... Their documentation uses `ds` pretty universally, so is it better to stick to that or the spelled out name?
https://docs.xarray.dev/en/stable/generated/xarray.Dataset.html

Reply:
I am annoyed that they chose to use `ds` instead of `dataset`, but I am totally fine with keeping it `ds`, `fname`, etc. if that is basically a 'standard' for the particular library.